includes/API.php

```diff
@@ -86,8 +86,23 @@ protected function do_post_parse_response_validation() {
 				throw new API\Exceptions\Request_Limit_Reached( $message, $response->get_api_error_code() );
 			}
 
+			/**
+			 * Handle invalid token errors
+			 *
+			 * @link https://developers.facebook.com/docs/graph-api/using-graph-api/error-handling#errorcodes
+			 */
+			if ( in_array( $response->get_api_error_code(), [ 10, 102, 190 ], false ) ) {
```

Comment:
From that table, 10 and 200-299 look like they may mean the same thing. Should we included 200-299 here?

Reply:
Probably a good idea! Added 62b2a56face6a4382942192e50e874e05e829ad4